deployer/deployer.py

```diff
@@ -506,3 +517,110 @@ def deploy(self, environment='pre', eureka=True, health_check=None, health_check
 
         if environment != 'pro':
             self.set_scheduler(version)
+
+    def deploy_elb(self, health_check='healthcheck', health_check_port=8000):
+        self.elbs = [self.app]
+
+        elb_data = self.get_or_create_loadbalancer_data(health_check, health_check_port)
+
+        self.validate_loadbalancer(elb_data, health_check, health_check_port)
+
+        if self.elb_dns:
+            domain_name = elb_data["loadBalancer"]["DNSName"]
+            self.check_or_create_route53(domain_name)
+
+    def get_or_create_loadbalancer_data(self, health_check, health_check_port):
+        elb_data = self.get_loadbalancer_data()
+
+        if elb_data is None:
+            if not self.create_loadbalancer(health_check, health_check_port):
+                print("load balancer creation failed", file=sys.stderr)
+                exit(1)
+            elb_data = self.get_loadbalancer_data()
+
+        return elb_data
+
+    def validate_loadbalancer(self, elb_data, health_check, health_check_port):
+        elb_healthcheck_target = elb_data.get("loadBalancer", {}).get("healthCheck", {}).get("target")
+
+        if elb_healthcheck_target is None:
+            raise Exception("Health check configuration not found on ELB {0}".format(elb_data["name"]))
+
+        search_data = re.search(r"^HTTP:{0}/*{1}$".format(health_check_port, health_check), elb_healthcheck_target)
+        if search_data is None:
+            wanted_healthcheck = "HTTP:{0}/{1}".format(health_check_port, health_check)
+
+            raise Exception("Deployed ELB healthcheck changed, Updates should be done manually:\n" +
+                            "Found: {0} | Requested {1}".format(elb_healthcheck_target, wanted_healthcheck))
+
+    def check_or_create_route53(self, domain_name):
+        hosted_zone = self.get_hosted_zone()
+
+        hosted_zone_id = hosted_zone["id"]
+        hosted_zone_name = hosted_zone["name"]
+
+        target_route_name = "{0}.{1}".format(self.app, hosted_zone_name)
+
+        route53 = self.get_route53(hosted_zone["id"], target_route_name)
+
+        if route53 is None:
+            self.create_route53_cname(hosted_zone_id, target_route_name, domain_name)
+
+        print("Service is exposed on " + target_route_name)
+
+    def get_hosted_zone(self):
+        r = self.request("hostedZone/list.json")
+
+        if r.status_code is not 200:
+            raise Exception("There isn't hosted zones on AWS")
+
+        json_response = r.json()
+        hosted_zones = [route for route in json_response if route.get("name", "").endswith("spain.schibsted.io.")]
```

Comment:
@padilo this is a open source repo, so that we need abstract if from our customer logic with the stepper, I think it's better option crate the route53 for the ELB only if a parameter is passed, and add parameters for our customer logic.